docs/source/querying/querybuilder/append.rst

```diff
@@ -749,7 +748,6 @@ You can also limit the number of rows returned with the method *limit*::
     qb = QueryBuilder()
     qb.append(
         CalcJobNode,
-        filters={'ctime':{'>': now - timedelta(days=3)}},
```

Comment:
Has this filter been removed for all examples? If not, I guess it should be?

Reply:
Well, not everywhere. If the description explicitly says
> How, can we have multiple filters? Suppose you are interested in all calculations in your database that are in state ‘FINISHED’ and were created **in the last n days**:

Then it should be kept.